src/models.py

```diff
@@ -175,11 +175,36 @@ def upgrade_onethreeoh(mod):
     )
 
 
+def upgrade_onethreeoh_to_onethreeone(mod):
+    "Upgrade LPCG model from 1.3.0 to version 1.3.1."
+    mm = aqt.mw.col.models
+    mm.addField(mod, mm.newField("Author"))
+
+    mod['css'].replace('.title', '.title, .author')
```

Comment:
I'm not sure why, but this replacement did not work in my test and the author was rendered left-aligned and normal size. Please take a look at what's going on there.

Reply:
Also, thoughts on making the source text `.title {` rather than just `.title`? My intuition says this will be less likely to do a replacement resulting in an invalid CSS rule if the user has edited the styling themselves, but I could imagine arguments for the other way as well...

Reply:
Oh, I know what the problem is. `.replace()` doesn't modify the string in-place, it returns the new value.

```suggestion
    mod['css'] = mod['css'].replace('.title', '.title, .author')
```

I've been using Python for 10 years and this one still gets me...

Reply:
Thank you for the insight here! `replace` tricked me as well.